eth/stagedsync/stage_execute.go

```diff
@@ -588,6 +588,7 @@ func recoverCodeHashPlain(acc *accounts.Account, db kv.Tx, key []byte) {
 
 func PruneExecutionStage(s *PruneState, tx kv.RwTx, cfg ExecuteBlockCfg, ctx context.Context, initialCycle bool) (err error) {
 	logPrefix := s.LogPrefix()
+	depositeAddress := params.Eth2DeployBlockNumber(cfg.chainConfig)
```

Comment:
I see on left side "Address" and on right side "blockNumber". Probably it's not address?